lib/create-append-anything/append-menu/AppendContextPadProvider.js

```diff
@@ -51,7 +51,7 @@ AppendContextPadProvider.prototype.getContextPadEntries = function(element) {
     return {
       'append': {
         group: 'model',
-        imageUrl: appendIcon,
+        html: `<div class="entry"> ${appendIcon}</div>`,
```

Comment:
@smbea is this extra space needed?

Reply:
Did not catch that. Fixed it via https://github.com/bpmn-io/bpmn-js-create-append-anything/pull/9/commits

Reply:
Thanks for following up!